src/gen_shaders.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+#Generates temporary hack shaders for previewing changes without touching .map
+import os
+
+template = \

```

Comment:
Feel free to completely overwrite our `.shader` file here, those textures are not in a `tex-` shared packages and it would be a mistake for a map to rely on those textures.

Note that we have a tool for generating `.shader` files from image files based on patterns ([Sloth](https://github.com/DaemonEngine/Sloth)). I may help you setting it up.